Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/SharedLogic/MessageBuilder.hs

```diff
@@ -73,3 +81,60 @@ buildSendAlternateNumberOTPMessage merchantId req = do
     merchantMessage.message
       & T.replace (templateText "otp") req.otp
       & T.replace (templateText "hash") req.hash
+
+data BuildSendEndRideMessageReq = BuildSendEndRideMessageReq
+  { rideAmount :: Text,
+    rideId :: Text
+  }
+  deriving (Generic)
+
+buildSendEndRideMessage :: (EsqDBFlow m r, CacheFlow m r) => Id DM.Merchant -> BuildSendEndRideMessageReq -> m Text
+buildSendEndRideMessage merchantId req = do
+  merchantMessage <-
+    QMM.findByMerchantIdAndMessageKey merchantId DMM.SEND_END_RIDE_MESSAGE
+      >>= fromMaybeM (MerchantMessageNotFound merchantId.getId (show DMM.SEND_END_RIDE_MESSAGE))
+  return $
+    merchantMessage.message
+      & T.replace (templateText "rideAmount") req.rideAmount
+      & T.replace (templateText "rideId") req.rideId
```

Comment:
It should be rideShortId instead of rideId.

Reply:
While building the request, I am assing Short id in the rideId

Reply:
renamed it to rideShortId